src/main/java/edu/illinois/cs/dt/tools/utility/iFixPlusPlugin.java

```diff
@@ -0,0 +1,744 @@
+                        errorMsg =
+                            "Module is not using a supported test framework (probably not JUnit), " +
+                            "or there is no test.";
+                    } else {
+                        errorMsg = "dt.detector.forceJUnit4 is true but no JUnit 4 runners found. Perhaps the project only contains JUnit 5 tests.";
+                    }
+                    TestPluginPlugin.mojo().getLog().info(errorMsg);
+                    return;
+                }
+            } else {
+                String errorMsg;
+                if (runners.size() == 0) {
+                    errorMsg =
+                        "Module is not using a supported test framework (probably not JUnit), " +
+                        "or there is no test.";
+                } else {
+                    // more than one runner, currently is not supported.
+                    errorMsg =
+                        "This project contains both JUnit 4 and JUnit 5 tests, which currently"
+                        + " is not supported by iDFlakies";
+                }
+                TestPluginPlugin.mojo().getLog().info(errorMsg);
+                return;
+            }
+        }
+        final Runner runner = InstrumentingSmartRunner.fromRunner(runners.get(0));
+
+        replayPath = Paths.get(Configuration.config().getProperty("replay.path"));
+        replayPath2 = Paths.get(Configuration.config().getProperty("replay.path2"));
+        dtname= Configuration.config().getProperty("replay.dtname");
+        output= Configuration.config().getProperty("replay.output");
+        slug= Configuration.config().getProperty("replay.slug");
+        subxmlFold = Configuration.config().getProperty("replay.subxmlFold");
+        rootFold = Configuration.config().getProperty("replay.rootFold");
+        tmpfile = Configuration.config().getProperty("replay.tmpfile");
+        module = Configuration.config().getProperty("replay.module");
+        diffFieldsFold = Configuration.config().getProperty("replay.diffFieldsFold");
+        subdiffsFold = Configuration.config().getProperty("replay.subdiffsFold");
+        reflectionFold = Configuration.config().getProperty("replay.reflectionFold");
+
+        int xmlFileNum = countDirNums(subxmlFold);
+        System.out.println("xmlFileName: " + xmlFileNum);
+
+        if (runner != null && module.equals(PathManager.modulePath().toString())) {
+            System.out.println("replyPath: " + replayPath);
+            System.out.println("module: " + module);
+
+            try {
+                Files.write(Paths.get(output),
+                        (lastPolluter() + ",").getBytes(),
+                        StandardOpenOption.APPEND);
+                // phase 0 check json file
+                System.out.println("~~~~~~ Begin to check if this test is a true order-dependent test.");
+                Configuration.config().properties().setProperty("statecapture.phase", "check");
```

Comment:
Does configuring the phase to be "check" affect anything for the testrunner run itself?